src/tests/ftest/get_remote_files.sh

```diff
@@ -130,7 +130,7 @@ list_tag_files() {
 #######################################
 compress_files() {
     # shellcheck disable=SC2086
-    find ${1} -maxdepth 0 -type f -size +1M -print0 | xargs -r0 lbzip2 -v
+    find ${1} -maxdepth 0 -type f -size +1M -print0 | sudo xargs -r0 lbzip2 -v
```

Comment:
Do we know why this is only needed sometimes though?

Reply:
No.  I've noticed `fchown` errors reported against `daos_control.log` and `daos_server_0.log` files - not sure if there are others.  It is not always both of those files at the same time however.

I believe we are seeing this now because we changed the order of compressing log files.  Before we would copy (`scp`) the log files back to the test launch node (presumably ensuring the files where owned by the correct user) before we compressed the files on the launch node.  Now we compress the files before copying them to the test launch node.

Reply:
Right.  I understand that we are now seeing this because we are not compressing a local copy of the file.  What I'm wondering is why the file fails a chown on the remote compression.  Who owns the file and what are it's permissions.  Contrast to when this error doesn't happen.

Is this a result of running servers as root perhaps?

Reply:
I did encounter the fchown error in local testing with a test that is using the standard way of running servers.  If the commands our standard way of launching servers uses root behind the scenes somewhere I would think the fchown issue would be consistently seen across at least the same set of tests and log files.

Reply:
> I did encounter the fchown error in local testing

Interesting.  It would be interesting to know the owner/group/permissions from the file and it's directory when this happens.

> with a test that is using the standard way of running servers. If the commands our standard way of launching servers uses root behind the scenes somewhere I would think the fchown issue would be consistently seen across at least the same set of tests and log files.

Agreed.  Which makes figuring out why it happens intermittently all that more important or at least interesting.  We could be papering over another/real issue by simply throwing the `sudo` hammer at it.  IMHO.

Reply:
I've attempted to hit the issue again manually and have only come across it once in numerous attempts (unfortunately with a `ls -a1` typo instead of `ls -al` in the debug code).  I can continue to use the debug code in my manual testing an open an issue if I run across it again with more debug information.

Reply:
I did hit the issue today with the debug code enabled:
```
    wolf-74: rc=0, output:                                                                                                                                                          
      Logging get_remote_files.sh output to /tmp/get_remote_files_20201130_145418.log                                                                                               
      Running /usr/lib/daos/TESTING/ftest/cart/cart_logtest.py ...                                                                                                                  
      Compressing files larger than 1M ...                                                                                                                                          
        Error compressing at least one of the following files in /var/tmp/daos_testing/*.log*:                                                                                      
      -rw-rw-r-- 1 pahender pahender         0 Nov 30 14:54 /var/tmp/daos_testing/test_reintegrate_basic_daos_admin.cart_logtest.log                                                
      -rw-rw-r-- 1 root     daos_admins   2219 Nov 30 14:53 /var/tmp/daos_testing/test_reintegrate_basic_daos_admin.log                                                             
      -rw-rw-r-- 1 pahender pahender         0 Nov 30 14:54 /var/tmp/daos_testing/test_reintegrate_basic_daos_control.cart_logtest.log
      -rw------- 1 pahender pahender     78633 Nov 30 14:54 /var/tmp/daos_testing/test_reintegrate_basic_daos_control.log.bz2
      -rw-rw-r-- 1 pahender pahender       932 Nov 30 14:54 /var/tmp/daos_testing/test_reintegrate_basic_daos_server.cart_logtest.log
      -rw-rw-r-- 1 pahender pahender    146231 Nov 30 14:54 /var/tmp/daos_testing/test_reintegrate_basic_daos_server.log.bz2
        Retrying ...
```